transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystemRootResourceDefinition.java

```diff
@@ -344,18 +346,47 @@ protected void finishModelStage(final OperationContext context, final ModelNode
 
             assert !USE_JOURNAL_STORE.isAllowExpression() && !USE_JDBC_STORE.isAllowExpression() : "rework this before enabling expression";
 
-            if (attributeName.equals(USE_JOURNAL_STORE.getName()) || attributeName.equals(USE_JDBC_STORE.getName())) {
-                if (newValue.asBoolean() == true) {
-                    // check the value of the mutual attribute and disable it if it is set to true
-                    final String mutualAttributeName = attributeName.equals(USE_JDBC_STORE.getName())
-                            ? USE_JOURNAL_STORE.getName()
-                            : USE_JDBC_STORE.getName();
-
-                    ModelNode resourceModel = model.getModel();
-                    if (resourceModel.hasDefined(mutualAttributeName) && resourceModel.get(mutualAttributeName).asBoolean()) {
-                        resourceModel.get(mutualAttributeName).set(new ModelNode(false));
-                    }
```

Comment:
I see. I expected alternatives validation to be much older. I will then cancel setAlternatives() in favour of this.